src/setup/rgw_setup

```diff
@@ -34,6 +37,21 @@ class SetupCmdBase(Cmd):
         parser.add_argument('--services', default='services', help='services')
         parser.add_argument('--config', default='config_url', help='config')
 
+    def _load_config(self, config_url: str):
+        """Load Config url."""
+        self.conf = MappedConf(config_url)
+        return self.conf
+
+    def _initialize_logging(self, conf: MappedConf):
+        """Initialize Logging."""
+        log_path = conf.get(LOG_PATH_KEY)
+        if log_path is None:
+            raise SetupError(errno.EINVAL, 'Log path is None.')
+        rgw_log_path = os.path.join(log_path, 'rgw', Conf.machine_id)
+        os.makedirs(rgw_log_path, exist_ok=True)
+        Log.init(SERVICE_NAME, rgw_log_path)
```

Comment:
Can you check and confirm that mini provisioner from other components print logs onto the screen as well. For this Log.init() needs to be passed a parameter

Reply:
Yes other MP interfaces prints log onto screen, Passed 'console_output=True' to Log.init() for printing logs onto screen.